src/stcal/ramp_fitting/likely_fit.py

```diff
@@ -62,22 +47,48 @@ def likely_ramp_fit(ramp_data, readnoise_2d, gain_2d):
 
     readtimes = get_readtimes(ramp_data)
 
+    # Check for a zeroframe
+    if ramp_data.zeroframe is not None:
+        # Read time for the zeroframe is the first entry in the first group
+        rt_zero = readtimes[0][0]
+
+        # Remove the zero readtime from the first group
+        readtimes[0] = readtimes[0][1:]
```

Comment:
If the zero frame is downlinked separately, the first group should always have >1 reads, but maybe we want to catch an exception to the above with an error message that the zero frame exists *and* the first element of readtimes does not have length >1?